speed-dreams/source-2.2.3/data/data/indicators/config.xml

```diff
@@ -70,5 +70,22 @@
                 <attstr name ="source" val="warning_brake_brake.wav"/>
             </section>
         </section>
+		<section name ="accelerate">
+			<section name="texture">
+				<attstr name ="source" val="Intervention_Brake_Proto_256.png"/>
+				<attnum name ="xpos"   val="0.75"/>
+				<attnum name ="ypos"   val="0.03"/>
+			</section>
+			<section name="text">
+				<attstr name ="content"   val="Brake"/>
+				<attnum name ="xpos"      val="0.75"/>
+				<attnum name ="ypos"      val="0"/>
+				<attstr name ="font"      val="jura-normal.glf"/>
+				<attnum name ="font-size" val="300"/>
+			</section>
+			<section name="sound">
+				<attstr name ="source" val="warning_brake_brake.wav"/>
+			</section>
+		</section>
```

Comment:
Don't use actual tabs, replace them with spaces. (I advice you to just set this up so your IDE automatically replaces tabs by 4 spaces)